progress/meter.go

```diff
@@ -58,15 +105,6 @@ func (p *ProgressMeter) Start() {
 	}
 }
 
-// Add tells the progress meter that a transferring file is being added to the
-// TransferQueue.
-func (p *ProgressMeter) Add(name string) {
-	idx := atomic.AddInt64(&p.transferringFiles, 1)
-	p.fileIndexMutex.Lock()
-	p.fileIndex[name] = idx
-	p.fileIndexMutex.Unlock()
-}
-
 // Skip tells the progress meter that a file of size `size` is being skipped
 // because the transfer is unnecessary.
 func (p *ProgressMeter) Skip(size int64) {
```

Comment:
Well, we were actually talking about the `Add() -> StartTransfer()` rename so it's consistent with `FinishTransfer()`. However, this is a valid point. I don't like any of the method names with `Estimate` TBH, and was hoping to come up with a better name. Maybe just `Add()` and `Skip()`, even if this new `Add()` is very different from the old one (that is now `StartTransfer()`)?